Stripe/Info.plist

```diff
@@ -7,7 +7,7 @@
 	<key>CFBundleExecutable</key>
 	<string>Stripe</string>
 	<key>CFBundleIdentifier</key>
-	<string>com.stripe.stripe-ios</string>
+	<string>$(PRODUCT_BUNDLE_IDENTIFIER)</string>
```

Comment:
actually, looks like this wasn't working – our manual installation test was catching the issue. i looked into it for a bit, but timed out and decided to just revert the change, and remove the unused PRODUCT_BUNDLE_IDENTIFIER from the xcconfig.